recipes/libiconv/all/conanfile.py

```diff
@@ -46,7 +46,7 @@ def build_requirements(self):
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         archive_name = "{0}-{1}".format(self.name, self.version)
```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
```

Reply:
Could you explain how the suggestion is relevant to the fix, please?

Reply:
It is the latest convention and simplifies several steps. When you are already motivated to change `source()` in so many recipes, it would be more useful to go all the way, instead of having to mass edit the same recipes again later

Reply:
Got it, thanks!